.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,47 @@
+name: Publish Docs
+
+on:
+  push:
+    tags:
+      # Only runs for version tags:
+      # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-of-specific-tags-occurs
+      - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
We can relax this if tags are only created for releases.

Reply:
So far tags are only created for releases, but I like that you made it very specific here :-)